### ISSUE TITLE
Other CNN Models

### ISSUE THREAD

**Issue by ahkarami** [huanghoujing/person-reid-triplet-loss-baseline#1]
Dear @huanghoujing,
Have you ever trained other ConvNets (_e.g._, ResNet101, ResNet152, ResNext, Inception-ResNet-V2, DenseNet) as the backbone of your code?

**Comment by huanghoujing** [comment]
Hi, I only tried ResNet-50. It's easy to replace the model with yours in file `tri_loss/model/Model.py`. Your model has to return feature with shape `[batch_size, out_channel]` to make it consistent with the training script.

**Comment by ahkarami** [comment]
Dear @huanghoujing,
Thank you very much for your answer.

**Comment by huanghoujing** [comment]
Thanks for attention : )
